<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [Default suite]" guice-stage="DEVELOPMENT">
  <test thread-count="5" name="Default test(failed)">
    <classes>
      <class name="calc.testCases.CalcDemoTC.CalcTest">
        <methods>
          <include name="invokeWSTestcase"/>
          <include name="afterMethod"/>
          <include name="VerifyCalcOperations" invocation-numbers="4"/>
          <include name="afterBaseClass"/>
          <include name="setSuiteSetUp"/>
          <include name="beforeBaseClass"/>
          <include name="afterSuite"/>
        </methods>
      </class> <!-- calc.testCases.CalcDemoTC.CalcTest -->
    </classes>
  </test> <!-- Default test(failed) -->
</suite> <!-- Failed suite [Default suite] -->
